### ISSUE TITLE
Clarify supported browsers

### ISSUE THREAD

**Issue by TomokiMiyauci** [denoland/deno_std#2053]
### Discussed in https://github.com/denoland/deno_std/discussions/2052

<div type='discussions-op-text'>

<sup>Originally posted by **TomokiMiyauci** March 22, 2022</sup>
```
// This module is browser compatible.
```

What does the comment browser compatible refer to?

Some modules use ES2022 features.
For example, [`Object#hasOwn`](https://github.com/denoland/deno_std/blob/main/collections/deep_merge.ts#L8).

According to MDN, this is not supported in some browsers; Safari 15.4+.

If "browser compatible" refers to the syntax
I think the documentation should clearly state that polyfills are required or specify which browsers support them.</div>

It is helpful to specify supported browsers somewhere in the document.

**Comment by fxrysh** [comment]
I guess it meant that it doesn't depend on `Deno.*` specific API which means you can transpile it down to ES6 (or other) and use it normally.

**Comment by TomokiMiyauci** [comment]
@fxrysh 
> I guess it meant that it doesn't depend on `Deno.*` 

Yes.This is clearly stated in the [documentation](https://deno.land/manual/contributing/style_guide#std).

**Comment by lino-levan** [comment]
@TomokiMiyauci I feel like this has been clarified. Is there a reason this issue shouldn't be closed?

**Comment by TomokiMiyauci** [comment]
@lino-levan   The maintainer believes that the supporting browsers need to be documented. #2052 

This work does not appear to have been done yet, have I missed something?

**Comment by lino-levan** [comment]
@TomokiMiyauci You've linked [where it is documented](https://deno.land/manual@v1.27.0/contributing/style_guide#std) yourself? Are you suggesting that this is put elsewhere as well?

**Comment by TomokiMiyauci** [comment]
@lino-levan I have modified the issue.

**Comment by iuioiua** [comment]
Can this be closed now, as we mark supported browsers in JSR?
